src/SFA.DAS.AANHub.Application.UnitTests/Common/Validators/RequestedByMemberId/RequestedByMemberIdValidatorTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿using FluentValidation.TestHelper;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.AANHub.Application.Common.Validators.RequestedByMemberId;
+using SFA.DAS.AANHub.Domain.Entities;
+using SFA.DAS.AANHub.Domain.Interfaces.Repositories;
+
+namespace SFA.DAS.AANHub.Application.UnitTests.Common.Validators.RequestedByMemberId;
+
+[TestFixture]
+public class RequestedByMemberIdValidatorTests
+{
+    [TestCase(null, false)]
+    [TestCase("00000000-0000-0000-0000-000000000000", false)]
+    [TestCase("1e6a97dd-8162-41d2-ba78-7d893111efe1", false)]
+    [TestCase("B46C2A2A-E35C-4788-B4B7-1F7E84081846", true)]
+    public async Task Validate_RequestedByMemberId_Exists(Guid? requestedByMemberId, bool isValid)
+    {
+        var command = new RequestedByMemberIdCommand() { RequestedByMemberId = requestedByMemberId };
+
+        var membersReadRepositoryMock = new Mock<IMembersReadRepository>();
+        var member = isValid ? new Member() : null;
+        membersReadRepositoryMock.Setup(m => m.GetMember(requestedByMemberId)).ReturnsAsync(member); ;
+        var sut = new RequestedByMemberIdValidator(membersReadRepositoryMock.Object);
+
+        var result = await sut.TestValidateAsync(command);
+
+        if (isValid)
+            result.ShouldNotHaveValidationErrorFor(c => c.RequestedByMemberId);
+        else
+            result.ShouldHaveValidationErrorFor(c => c.RequestedByMemberId);
+
+    }
+
+    public class RequestedByMemberIdCommand : IRequestedByMemberId
```

Comment:
If this class is for test purposes only, it should be private and probably include "Test" in the name, or something